toonz/sources/toonz/castviewer.cpp

```diff
@@ -752,11 +752,12 @@ QMenu *CastBrowser::getContextMenu(QWidget *parent, int index) {
 
   menu->addAction(cm->getAction(MI_ExposeResource));
   menu->addAction(cm->getAction(MI_ShowFolderContents));
-  menu->addAction(cm->getAction(MI_ViewFile));
+  if(!audioSelected && !paletteSelected)
+	  menu->addAction(cm->getAction(MI_ViewFile));
   menu->addAction(cm->getAction(MI_FileInfo));
-  if (audioSelected && !paletteSelected && !vectorLevelSelected &&
-      !otherFileSelected)
-    return menu;
+//  if (audioSelected && !paletteSelected && !vectorLevelSelected &&
+//      !otherFileSelected)
+//    return menu;
```

Comment:
Thanks for this.  I think these commented lines can be fully removed.

Reply:
Done. Thanks for reviewing.